src/cryptocom/exchange/api.py

```diff
@@ -24,16 +27,63 @@ def __init__(
             auth_required=True, timeout=25, retries=6,
             root_url='https://api.crypto.com/v2/',
             ws_root_url='wss://stream.crypto.com/v2/', logger=None):
+
         self.api_key = api_key
         self.api_secret = api_secret
         self.root_url = root_url
         self.ws_root_url = ws_root_url
         self.timeout = timeout
         self.retries = retries
+        self.last_request_path = ''
+        self.limits = {
+            # method: (req_limit, period)
+            'private/create-order': (14, 0.1),
+            'private/cancel-order': (14, 0.1),
+            'private/cancel-all-orders': (14, 0.1),
+            'private/margin/create-order': (14, 0.1),
+            'private/margin/cancel-order': (14, 0.1),
```

Comment:
why not create here limiter per each tuple

self.rate_limiters = {url: RateLimiter(*args) for url, args in self.limits}

Reply:
instead of one url per key you can use multiple in tuple ('private/create-order', 'private/cancel-order'): (14, 0.1)